### ISSUE TITLE
Primitive instances inheriting attributes of other primitive instances in scene

### ISSUE THREAD

**Issue by jcarpenter** [aframevr/aframe#261]
Any primitive instance that does not manually declare it’s own attribute value (eg `radius="2"`) will automatically inherit the value of that attribute if it is declared on another primitive instance in the scene.

For example:

``` html
<!-- Orange -->
<a-sphere position="-2 4 -5" radius="0.5" color="#F16745"></a-sphere>

<!-- Green -->
<a-sphere position="2 4 -5" color="#7BC8A4"></a-sphere>
```
Expected result is the green sphere renders with `radius="1"`, which is default (as specified in the primitive definition). Instead it inherits the `0.5` value of the orange sphere.

![screen shot 2015-12-05 at 5 09 07 pm](https://cloud.githubusercontent.com/assets/1425497/11611152/25bdde16-9b73-11e5-8c59-afa99dc4fb6a.png)

**Comment by cvan** [comment]
@ngokevin: can you confirm this is fixed in PR #883?

**Comment by ngokevin** [comment]
This is fixed in the PR from the initial implementation on josh's branch, but I'm closing this issue since the new primitives never landed anywhere.
